src/components/EmptyResults.vue

```diff
@@ -0,0 +1,27 @@
+<template>
+  
+  <div class="col text-center no-found">
+    <h2 class="text-light font-weight-light">
+      No films found
+    </h2>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css"
```

Comment:
move in global styles, not nessesary imprort in every component

also vue doesn't like style script inside
```
<div id="vue-app">
   // bad bad 
   <style
   <script
</div>
```